cpp/proto/xjson_serializer.cc

```diff
@@ -3,6 +3,7 @@
 
 #include <glog/logging.h>
 #include <math.h>
+#include <objecthash.h>
```

Comment:
Yeah! (Though if you prefer one massive PR with usage too that can probably be arranged :) )